archivers/xz/Portfile

```
@@ -7,10 +7,10 @@ PortGroup       github 1.0
 epoch           1
 # 5.6.0 and 5.6.1 contained Linux-specific malware. 5.6.2 and later are OK.
 # https://tukaani.org/xz-backdoor/
-github.setup    tukaani-project xz 5.4.7 v
-checksums       rmd160  b78831a070fde3715fdac3a509c6a66bd8f87a93 \
-                sha256  9976ed9cd0764e962d852d7d519ee1c3a7f87aca3b86e5d021a45650ba3ecb41 \
-                size    2166519
+github.setup    tukaani-project xz 5.6.2
```

The tag suffix must be retained.
```suggestion
github.setup    tukaani-project xz 5.6.2 v
```